pinot-tools/src/main/resources/examples/stream/airlineStats/airlineStats_realtime_table_config.json

```diff
@@ -25,8 +25,7 @@
       "stream.kafka.zk.broker.url": "localhost:2191/kafka",
       "stream.kafka.broker.list": "localhost:19092",
       "realtime.segment.flush.threshold.time": "3600000",
-      "realtime.segment.flush.threshold.size": "50000",
-      "stream.kafka.consumer.prop.auto.offset.reset": "smallest"
+      "realtime.segment.flush.threshold.size": "50000"
```

Comment:
why remove this?

Reply:
I thought I had modified this for testing. looks like I accidentally deleted the offset reset config too. tks for catching this.